services/blockchain-connector/shared/sdk/formatter.js

```diff
@@ -133,6 +133,8 @@ const formatEvent = (event) => {
 	const eventData = eventDataSchema
 		? codec.decodeJSON(eventDataSchema, Buffer.from(event.data, 'hex'))
 		: { data: event.data };
+	// TODO: Remove this after SDK exposes all event schemas
+	if (!eventDataSchema) console.warn(`Event data schema missing for ${event.module}:${event.name}`);
```

Comment:
Is there any reason that we are using `console` instead of `logger` from `lisk-service-framework`

Reply:
This is temporary to just keep us notified about the missing event schemas and will be remove soon.